### PR TITLE
Merge code-splitting-{import,require}

### LINKED ISSUE #992: Merge code-splitting-{import,require}
Idea from #988. Possible title: "Code Splitting – async loading"

### PR DESCRIPTION
**Pull request by simon04**
Implements and fixes #992.

### CODE REVIEW

File: antwar.config.js [hunk 1]
```diff
@@ -67,6 +67,8 @@ module.exports = {
           /^\.\/.*\.md$/
         );
       }, {
+        'code-splitting-import': '/guides/code-splitting-import',
```
Comment (skipjack) [inline antwar.config.js]: @simon04 is this supposed to be:

```
'code-splitting-import': '/guides/code-splitting-async#import'
```

or something similar?

Comment (skipjack) [inline antwar.config.js]: Aside from that this PR lgtm 👍

Comment (simon04) [inline antwar.config.js]: Since the `import()` part starts right on the top (after a short introduction), I refrained from adding an anchor.

Comment (skipjack) [inline antwar.config.js]: @simon04 yeah but didn't you remove the `code-splitting-import` guide? Shouldn't the redirect be to `/guides/code-splitting-async`?

Comment (simon04) [inline antwar.config.js]: Ahh, right. Sure! I'll amend.

Comment (simon04) [inline antwar.config.js]: Done. Thanks for spotting this glitch!

**Comment by simon04** [comment]
Let's go for it. Funnily we got 3 PR's on this file within 16 hours.

**Comment by skipjack** [comment]
@simon04 hahaha yea I was thinking the same thing, plus I'm working on another PR to resolve `npm run test` issues (#1041) which will touch this file as well 😆 .

**Pull request merged**